Documentation/install/upgrade.rst

```diff
@@ -332,6 +332,11 @@ Annotations:
 
   - ``global.hubble.tls.enabled=false``
   - ``global.hubble.tls.auto.enabled=false``
+* Cilium has upgraded its CRDs to v1, from v1beta1. Users who are upgrading
+  from Kubernetes below v1.16.0 to past Kubernetes v1.16.0, while running
+  Cilium v1.9 or greater, should run the pre-flight checker mentioned above.
+  The pre-flight check will ensure the custom resources installed inside the
+  cluster are well-formed.
```

Comment:
I'd usually expect (and for that matter, encourage) separate upgrades of Cilium vs. Kubernetes. At a glance I found this a bit ambiguous as to whether this is a step we're trying to say users should run during Cilium upgrade or trying to give them a heads up if they're upgrading K8s...?

Maybe splitting out into a few questions:
* Do users on K8s 1.16 or higher have to take any action?
* Do users on k8s 1.15 or lower have to take any action?
* Is this action required during Cilium 1.9 upgrade?
* Is this action required during K8s version upgrade?

Reply:
I think the intent of this note is to be a heads up to users who are considering upgrading their K8s clusters, after they've already upgraded to Cilium 1.9. 

To answer your questions, users do not have to take any additional action if they are keeping their K8s version the same. If users are upgrading their Kubernetes version greater than and including 1.16, the recommended action is to upgrade to Cilium 1.9 first. Then run the preflight checker, apply any recommendations from that, then upgrade to later versions of K8s.